sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -533,7 +533,7 @@ export class ApplicationStudentsController extends BaseController {
           userToken.studentId,
```

Comment:
There's a code smell about duplicate block of code. Are you going to work on that?

Reply:
Planning to do it in the upcoming PR. Maybe further refactoring in that area because the recent changes generate more possible flow and it is more complex to follow right now.